src/components/ChatMode/index.vue

```diff
@@ -425,6 +466,28 @@ export default {
   position: relative;
 }
 
+/* Group this message with prev-message (no padding-top and modify border-radius) */
+#conversation-container .bubble-container.grp-with-prev-msg {
+  padding-top: 0;
+}
+#conversation-container .bubble-container.grp-with-prev-msg[data-side='bot'] .v-sheet {
+  border-top-left-radius: 5px;
+}
+#conversation-container .bubble-container.grp-with-prev-msg[data-side='user'] .v-sheet {
+  border-top-right-radius: 5px;
+}
+
+/* Group this message with next-message (no padding-bottom and modify border-radius) */
+#conversation-container .bubble-container.grp-with-next-msg {
+  padding-bottom: 0;
+}
+#conversation-container .bubble-container.grp-with-next-msg[data-side='bot'] .v-sheet {
+  border-bottom-left-radius: 5px;
+}
+#conversation-container .bubble-container.grp-with-next-msg[data-side='user'] .v-sheet {
+  border-bottom-right-radius: 5px;
+}
```

Comment:
Please target `.upil-text-bubble` instead of `.v-sheet` so that we don't accidentally modify users' custom components. Also, update this code with that change even though I wrote it :)

```css
#conversation-container >>> .v-sheet {
  border-radius: 15px;
}
```

Reply:
I fixed this. Please check it.